src/index.ts

```diff
@@ -445,6 +447,9 @@ export async function assemble(config: Configuration, components: Components): P
   // Backwards compatibility (data)
   await ensureBackwardsCompatibility(components, config)
 
+  // Event emitter
+  const eventEmitter = createEmitter()
```

Comment:
I wonder if we should add this is a `component`(dependency) so we don't need to pass it in as a separate arg to each function that uses it? 🤔 though I supposed that may expose it to developers within the context of their client app, which we maybe don't want? Sorry, I'm just thinking out loud 😅

Reply:
Hmm maybe, I don't see this needing to be a component, since event emitting is such a generic thing. We can always revise later, maybe I'll change my mind as we add more events.